pkg/fanal/image/daemon/image_test.go

```diff
@@ -115,6 +115,52 @@ func Test_image_ConfigNameWithCustomDockerHost(t *testing.T) {
 	assert.Nil(t, err)
 }
 
+func Test_image_ConfigNameWithCustomPodmanHost(t *testing.T) {
+
+	ref, err := name.ParseReference("alpine:3.11")
+	require.NoError(t, err)
+
+	eo := engine.Option{
+		APIVersion: opt.APIVersion,
+		ImagePaths: map[string]string{
+			"index.docker.io/library/alpine:3.11": "../../test/testdata/alpine-311.tar.gz",
+		},
+	}
+
+	var podmanSocket string
+
+	if runtime.GOOS != "windows" {
+		runtimeDir, err := os.MkdirTemp("", "daemon")
+		require.NoError(t, err)
+
+		dir := filepath.Join(runtimeDir, "image")
+		err = os.MkdirAll(dir, os.ModePerm)
+		require.NoError(t, err)
+
+		customDockerHost := filepath.Join(dir, "image-test-podman-socket.sock")
+		eo.UnixDomainSocket = customDockerHost
+		podmanSocket = customDockerHost
+	}
+
+	te := engine.NewDockerEngine(eo)
+	defer te.Close()
+
+	if runtime.GOOS == "windows" {
```

Comment:
Windows test is FAIL - https://github.com/aquasecurity/trivy/actions/runs/8145689974/job/22269334521?pr=6256#step:10:3751

Looks like we need to skip this test for windows (https://github.com/aquasecurity/trivy/blob/9361cdb7e28fd304d6fd2a1091feac64a6786672/pkg/fanal/image/daemon/podman_test.go#L45-L47).

Reply:
Hey, @DmitriyLewen, I need a clarification, what should happen if the test were run on Windows should the test pass by default?